### ISSUE TITLE
randomize the type of search - mobile and PC?

### ISSUE THREAD

**Issue by CSJudd** [sealemar/BingRewards#125]
is it possible to randomize the search mode to reduce "bot look"? Now we run all the mobile (or PC) at once, then switch to the other - instead randomize some of each the whole account session?  Getting a lot of Bans recently...  Would also be nice to "go under" the point cap, instead of always shooting over it - again to minimize the "bot look", have a negative range allowed in the maxsearches - so if we want to do 12 points instead of 15...

**Comment by amayer5125** [comment]
@CSJudd I get randomizing to a certin extent. But how many people acutally search something on a computer then a couple seconds later search on their phone then back to the computer? Maybe if we could group 5 or 10 searches together then switch.
I really like your less then max points idea tho. If you used that with the random extra searches you could get random numbers from say 18 points up to the 25 max. Over time this would make it much harder to be detected as bots.
Just curiouse how many accounts are you running on the IPs that are getter banned? And are you running them manually or through cron (or something similar)?

**Comment by CSJudd** [comment]
Yeah thinking more of the groups 5-10, not single or smaller for the random back/forth device switching

The less than/negative parameters would be wonderful! 

I manually run them at random points thru the day on a few machines (unique public IPs) no cron yet.  Using 4 and not the max 5 accounts.

Have also tried a few setups where it was "split" having 2 distinct "BingRewards" directories and having a small random sleep on each (up to 20 mins from "launch") to simulate appearance of a household using.

**Comment by bbenz3** [comment]
A previous issue I brought up had two parts, one was adding additional searches over the minimum and the second was to have a command line switch to only run mobile our desktop searches. I currently have mine setup with a batch file using five separate folders, one for each account. The batch file it's called by the cron job. I randomize the order that the

**Comment by iRobie** [comment]
This is off topic for the actual request: what about adding a random click to the results page? Not all the time, but randomly select a random link to click, and open that link? This is beyond my python ability.

I've also added the option to read reddit RSS feeds. I didn't test with any other RSS feeds, and haven't tested the changes beyond getting reddit to work. I've forked this code.
